### ISSUE TITLE
Feature Request: Change Chat Background

### ISSUE THREAD

**Issue by maciboy** [Neo11/ChitChat#39]
Would it be possible to change the in-app background to e.g. a pure white one or even a user-specified image?

(Thanks for this fork and the updates! Hope it keeps alive, since ChitChat is the only client not switching on my Nvidia graphics :/)

Update: some colorful patterns were added by WhatsApp ;)

**Comment by c-o-m-m-a-n-d-e-r** [comment]
Yes it is ... but have to write some more code for Settings Dialog etc.
